### DISCUSSION TITLE
In need of advice .

### DISCUSSION THREAD

**Discussion by ZahierRei** [Vdauphin/HeartsAndMinds#1626] [Question and answer] [answered]
Hi I'm in need of advice .

1) Currently if one dies or abort in game when they re-enter they spawn at their last location how do i make it they spawn fresh again

2) As we know in order to save the game we require an admin is it possible to add like a save point on an item so that when there no admin the players can still save the game ?

**Comment by Vdauphin** [comment]
Hello,

1. That might be a bug after you edit the mission, looks like you enable "INSTANT" https://community.bistudio.com/wiki/Arma_3:_Respawn
2. You have different option in mission parameters (https://github.com/Vdauphin/HeartsAndMinds/blob/master/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/def/param.hpp#L44-L73)
Or something like this may work:
```sqf
_action = ["savemygame", "Save", "", {[] remoteExecCall ["btc_db_fnc_save", 2]}, {true}] call ace_interact_menu_fnc_createAction;
[btc_gear_object, 0, ["ACE_MainActions"], _action] call ace_interact_menu_fnc_addActionToObject;
```
In init player file

Cheers

**Reply by ZahierRei** [reply]
as for No.1 I've check the description text its set to "BASE or ID 3"
![Respawn setting](https://github.com/Vdauphin/HeartsAndMinds/assets/175175598/87bd1176-916d-429d-bee8-cd9ab2fe8091)

as for No.2 do i copy this and add to the init in an item say like a data terminal or a Laptop in eden editor ?

![Save init](https://github.com/Vdauphin/HeartsAndMinds/assets/175175598/1da75d48-46b3-4a4b-92bb-9eaf4787b802)

**Reply by Vdauphin** [reply, marked as answer]
1. No idea then
2. then you want to replace the `btc_gear_object` with `this`
```sqf
 _action = ["savemygame", "Save", "", {[] remoteExecCall ["btc_db_fnc_save", 2]}, {true}] call ace_interact_menu_fnc_createAction;
[this, 0, ["ACE_MainActions"], _action] call ace_interact_menu_fnc_addActionToObject;
```

**Reply by ZahierRei** [reply]
i tried add the script to the item init as such . but it doesnt work . no save option pops out .

**Reply by Vdauphin** [reply]
I edited my answer 
Sorry for the mistake

**Reply by ZahierRei** [reply]
now it works thank you .
